ui/src/buckets/actions/index.ts

```diff
@@ -180,6 +185,9 @@ export const deleteBucket = (id: string, name: string) => async (
 
     dispatch(removeBucket(id))
     dispatch(checkBucketLimits())
+    dispatch(() => ({
+      type: 'PLERPS',
```

Comment:
? :D

Reply:
oops, ill pull that out.  thanks for spotting that.